README.md

```diff
@@ -136,6 +136,14 @@ Running
 ```
 will remove the cluster created in the preceding steps.
 
+### An example demo 
```

Comment:
Instead of a new section. I would actually put this at the top just before all of the commands, introducing the steps that are about to be taken.

Reply:
Moved it to the beginning of the `Demo` section.

Reply:
You didn't remove this section as part of the change.

Reply:
This still hasn't been removed.

Reply:
> This still hasn't been removed.

Deleted.